atm.module

```diff
@@ -83,16 +82,87 @@ function atm_theme($existing, $type, $theme, $path) {
  * Attach atm/js in page.
  */
 function atm_page_attachments(array &$attachments) {
-  $attachments['#attached']['library'][] = 'atm/api';
+  /** @var AtmApiHelper $helper */
+  $helper = \Drupal::service('atm.helper');
+
+  $routName = \Drupal::routeMatch()->getRouteName();
+
+  if ($routName == 'atm.config.page') {
+    $attachments['#attached']['library'][] = 'atm/api.admin';
+  }
+
+  if ($routName == 'atm.config.page' || $routName == 'entity.node.canonical') {
+    /** @var \Drupal\node\Entity\Node $node */
+    $node = \Drupal::routeMatch()->getParameter('node');
+
+    if ($routName == 'atm.config.page' || in_array($node->getType(), $helper->getSelectedContentTypes())) {
+      $atmApi = $helper->get('build_path') . '?' . microtime();
+
+      $https = &$_SERVER['HTTPS'];
```

Comment:
Please set https check to `!empty($_SERVER['HTTPS']) && $_SERVER['HTTPS'] !== 'off' || $_SERVER['SERVER_PORT'] == 443` this may fix issues on ISAPI with IIS